docs/authentication/social-connections/dropbox.mdx

```diff
@@ -29,10 +29,10 @@ First, you need to register a new OAuth Dropbox app at the [Dropbox App Console]
 
 First you need to choose the API type, the App's type of access and to set a name for your new application. You also need to add the **OAuth Redirect URLs.**
 
-In the Clerk Dashboard, go to **User & Authentication > [Social Connections](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)** and enable Dropbox. In the modal that opened, ensure **Use custom credentials** is enabled and copy **Authorized redirect URI**. Paste the value into the **OAuth Redirect URIs** input and click create.
+Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **Dropbox**. Toggle on **Use custom credentials** and copy **Authorized redirect URI**. Paste the value into the **OAuth Redirect URIs** input and click create.
 
 Once all the above are complete, copy the **Client ID** and **Client Secret.** Go back to the Clerk Dashboard and paste them into the respective fields.
 
 ![Copying values from the Dropbox dashboard](/docs/images/authentication-providers/dropbox/copying-values-from-dropbox-dashboard.jpg)
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with Dropbox is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with Dropbox is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with Dropbox is now configured for your instance.
```